client/lib/api/content/content_bundle.dart

```diff
@@ -120,3 +137,19 @@ class ContentBundleSchemaVersionException implements Exception {}
 
 /// Indicates an error interpreting the content bundle data according to the expected schema.
 class ContentBundleDataException implements Exception {}
+
+/// An open-ended map of control values that can be embedded into a content bundle.
+class ContentControls {
+  Map<String, String> map;
+
+  ContentControls(this.map);
+
+  String value(Object key, String defaultValue) {
+    return map[key.toString()] ?? defaultValue;
+  }
+
```

Comment:
You might want to use the general display condition functionality that exists in the codebase for this.